src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -209,15 +209,11 @@ public void onTraversalStart() {
 
     @Override
     public void apply(final AssemblyRegion region, final ReferenceContext referenceContext, final FeatureContext featureContext ) {
```

Comment:
You also have to think about the issue of sharing code across the walker and spark versions of tools -- for that purpose, static utility methods are by far the best approach.